GREEDY/App.config

```diff
@@ -9,6 +9,7 @@
   </startup>
   <appSettings>
     <add key="OcrLanguage" value="lit" />
+    <add key="LogPath" value="../../Log.txt" />
```

Comment:
../../ looks not really grate. Can you change it or after copy out you will get same issue as with tesseract tessdata?